File: src/main/java/client/database/data/PendingNameChanges.java
package client.database.data;

public class PendingNameChanges {
   private int id;

   private int characterId;

   private String oldName;

   private String newName;

   public PendingNameChanges(int id, int characterId, String oldName, String newName) {
      this.id = id;
      this.characterId = characterId;
      this.oldName = oldName;
      this.newName = newName;
   }

   public int getId() {
      return id;
   }

   public int getCharacterId() {
      return characterId;
   }

   public String getOldName() {
      return oldName;
   }

   public String getNewName() {
      return newName;
   }
}

File: engine/model/src/main/scala/client/database/data/PendingNameChanges.scala
package client.database.data

class PendingNameChanges(private var _id: Int, private var _characterId: Int, private var _oldName: String,
                         private var _newName: String) {
  def id: Int = _id

  def characterId: Int = _characterId

  def oldName: String = _oldName

  def newName: String = _newName
}
